### ISSUE TITLE
Error running with api-version=2: unknown flag: --api-version

### ISSUE THREAD

**Issue by debedb** [derekparker/delve#617]
1. Delve version:

$ dlv version
Delve Debugger
Version: 0.11.0-alpha
Build: v0.11.0

2. Go version:

$ go version
go version go1.6.2 darwin/amd64

3. What operating system and processor architecture are you using?

OSX 10.11.2
15.2.0 Darwin Kernel Version 15.2.0: Fri Nov 13 19:56:56 PST 2015; root:xnu-3248.20.55~2/RELEASE_X86_64 x86_64

4. What did you do?

Tried running this, as instructed by https://github.com/derekparker/delve/tree/master/Documentation/api :

`dlv debug --headless --api-version=2 --log --listen=127.0.0.1:8181`

5. What did you expect to see?

Building and debugging of the program.

6. What did you see instead?

`Usage:
  dlv debug [flags]

Global Flags:
      --build-flags="": Build flags, to be passed to the compiler.
      --headless[=false]: Run debug server only, in headless mode.
      --init="": Init file, executed by the terminal client.
  -l, --listen="localhost:0": Debugging server listen address.
      --log[=false]: Enable debugging server logging.

Error: unknown flag: --api-version`

**Comment by aarzilli** [comment]
My guess is that you need to update your delve

```
$ cd $GOPATH/src/github.com/derekparker/delve/
$ git pull
$ CERT=dlv-cert make install
```

**Comment by debedb** [comment]
Maybe - this was installed with https://github.com/derekparker/delve/blob/master/Documentation/installation/osx/install.md instructions (via homebrew). But the version seems to be the same as the latest version tag.

**Comment by aarzilli** [comment]
We haven't made a new release in a while.

**Comment by debedb** [comment]
I'll try when I get a chance, thanks. Maybe a new release is warranted, given at least this feature (new api version)?

**Comment by debedb** [comment]
That works, thanks

**Comment by RylanSchaeffer** [comment]
@aarzilli or whoever, how does one update their delve? I"m receiving the following error when I try to set the api version:

`unknown flag: --api-version`

My `dlv` looks out of date:

```
$dlv version
Delve Debugger
Version: 0.11.0-alpha
Build: 
```

But I can't figure out how to upgrade it. I've tried `go get -u github.com/go-delve/delve/cmd/dlv` and `go get -u github.com/derekparker/delve/cmd/dlv`, but neither worked. I also tried the suggestion to `make install`:

```
$ git clone https://github.com/go-delve/delve.git $GOPATH/src/github.com/go-delve/delve
$ cd $GOPATH/src/github.com/go-delve/delve
$ make install
```

But that appears to have no effect:

```
$ dlv version
Delve Debugger
Version: 0.11.0-alpha
Build:
```
